readme.md

```diff
@@ -12,15 +12,17 @@ const regularMap = new WeakMap();
 regularMap.set({}, true);
```

Comment:
This also needs the same treatment for consistency. It might be worth adding "const obj = {}" as well

Reply:
Missed this one. Added now, thanks!